nixos/tests/printing.nix

```diff
@@ -54,16 +56,16 @@ import ./make-test-python.nix (
     start_all()
 
     with subtest("Make sure that cups is up on both sides and printers are set up"):
-        server.wait_for_unit("cups.${if socket then "socket" else "service"}")
-        client.wait_for_unit("cups.${if socket then "socket" else "service"}")
+        server.wait_for_unit("ensure-printers.service")
+        client.wait_for_unit("ensure-printers.service")
 
     assert "scheduler is running" in client.succeed("lpstat -r")
 
     with subtest("UNIX socket is used for connections"):
         assert "/var/run/cups/cups.sock" in client.succeed("lpstat -H")
 
     with subtest("HTTP server is available too"):
-        client.succeed("curl --fail http://localhost:631/")
+        ${pkgs.lib.optionalString listenTcp ''client.succeed("curl --fail http://localhost:631/")''}
```

Comment:
```suggestion
        ${lib.optionalString listenTcp ''client.succeed("curl --fail http://localhost:631/")''}
```

Reply:
Done.  However, that requires to `inherit (pkgs) lib;` before.